docs/goals-canvas.md

```diff
@@ -5,7 +5,7 @@ title: Goals canvas
 
 # Goals canvas BRK bevragen
 
-## Zoek persoon
+## Zoek met persoon
```

Comment:
Is 'Zoek **met** persoon' inderdaad de juiste benaming, moet het niet alleen 'Zoek persoon' zijn?

Reply:
Via merge verwijderd